src/components/routes/Member/AuthenticatedMember/UpdateCredentialRoute/UpdateUsername/UpdateUsername.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { HEADER_TEXT } from '../../../../../../constants/common/commonConstants';
+import Form from '../../../../../common/Typography/Form';
+import Container from '../../../../../common/Typography/Container';
+import Alert from '../../../../../common/Alert';
+import MemberModule from '../../../../../../services/member';
+import Button from '../../../../../common/Typography/Button';
+
+const { updatePasswordAction } = MemberModule.actions;
+
+const UpdateUsername = (props) => {
+  const {
+    config: { updateUsernameForm },
+    memberDetails,
+    updatePasswordIsSuccess,
+    updatePasswordMessage,
+  } = props;
+
+  /**
+   * Submit handler for update password form
+   * @param {Object} formData
+   */
+  const onSubmit = ({ formData }) => {
+    props.updatePasswordAction(
```

Comment:
Can we rename this action to 
```suggestion
    props.updateMemberCredentialsAction(
```

Reply:
done.